integration_tests/bzlmod/src/sample.c++

```diff
@@ -0,0 +1,34 @@
+#include "sample.capnp.h"
+#include <capnp/message.h>
+#include <capnp/serialize-packed.h>
+#include <iostream>
+
+int main() {
+  ::capnp::MallocMessageBuilder message;
+
+  AddressBook::Builder addressBook = message.initRoot<AddressBook>();
+  ::capnp::List<Person>::Builder people = addressBook.initPeople(2);
+
+  Person::Builder alice = people[0];
+  alice.setId(123);
+  alice.setName("Alice");
```

Comment:
Integration tests actually build and use a cap'n'proto from C++ to keep things honest